mobile-save-for-later-user-deletion/conf/cfn.yaml

```diff
@@ -100,6 +100,7 @@ Resources:
     Type: AWS::SQS::Queue
     Properties:
       QueueName: !Sub UserIdDeleteQueue-${Stage}
+      VisibilityTimeout: 300
```

Comment:
We could consider reducing this timeout (and the corresponding lambda timeout). It typically seems to take ~25 seconds to execute. 

However, as far as I can tell this is used for some kind of background processing, so I don't think there's any particular advantage to reducing this.

Reply:
Without knowing why the visibility timeout was increased to 300s I agree that the safest thing (for now) is to define this value in the cloudformation, we can investigate if this can be reduced later